src/tests/io/test_loadtxt.f90

```diff
@@ -1,55 +1,63 @@
 program test_loadtxt
-use stdlib_experimental_kinds, only: int32, sp, dp
-use stdlib_experimental_io, only: loadtxt
-use stdlib_experimental_error, only: error_stop
-implicit none
+  use stdlib_experimental_kinds, only: int32, sp, dp
+  use stdlib_experimental_io, only: loadtxt
+  use stdlib_experimental_error, only: error_stop
+  implicit none
 
-integer(int32), allocatable :: i(:, :)
-real(sp), allocatable :: s(:, :)
-real(dp), allocatable :: d(:, :)
+  integer(int32), allocatable :: i(:, :)
+  real(sp), allocatable :: s(:, :)
+  real(dp), allocatable :: d(:, :)
+  complex(dp), allocatable :: z(:, :)
 
-call loadtxt("array1.dat", i)
-call print_array(i)
+  call loadtxt("array1.dat", i)
+  call print_array(i)
 
-call loadtxt("array1.dat", s)
-call print_array(s)
+  call loadtxt("array1.dat", s)
+  call print_array(s)
 
-call loadtxt("array1.dat", d)
-call print_array(d)
+  call loadtxt("array1.dat", d)
+  call print_array(d)
 
-call loadtxt("array2.dat", d)
-call print_array(d)
+  call loadtxt("array2.dat", d)
+  call print_array(d)
 
-call loadtxt("array3.dat", d)
-call print_array(d)
+  call loadtxt("array3.dat", d)
+  call print_array(d)
 
-call loadtxt("array4.dat", d)
-call print_array(d)
+  call loadtxt("array4.dat", d)
+  call print_array(d)
+
```

Comment:
```suggestion

  call loadtxt("array5.dat", c)
  call print_array(c)
  
```
I suppose it's ok to reuse the file "array5.dat" for both single and double complex?